src/networks/evm/client.ts

```diff
@@ -1,24 +1,26 @@
 import Web3 from 'web3';
 import BN from 'bn.js';
 import { Contract } from 'web3-eth-contract';
-import { AbiItem } from 'web3-utils';
-import { provider } from 'web3-core';
+import { AbiItem, isAddress } from 'web3-utils';
+import { TransactionReceipt, provider } from 'web3-core';
 import { TransactionConfig } from 'web3-core';
-
 import { TxFee, TxStatus } from '../../networks/transaction';
 import { convertTransaction } from './utils';
```

Comment:
It looks like this function is not necessary now so we can drop `utils.ts`

Reply:
Removed in https://github.com/zkBob/zeropool-support-js/pull/9/commits/93dcd48ebeb76a407195e39fc31039989d0eb9fa